epub34/authoring/index.html

```diff
@@ -1184,6 +1184,13 @@ <h3>Exempt resources</h3>
 					resources=]. It is most similar to a [=foreign resource=] in that it is not guaranteed [=reading
 					system=] support, but, like a core media type resource, does not require a fallback.</p>
 
+				<p>In some cases, the exemptions defined in this section overlap with how existing core media type
+					resources are included in [=EPUB content documents=] (e.g., using the [[html]] <a
+						data-cite="html#the-link-element"><code>link</code></a> element to include CSS style sheets,
+					importing fonts into CSS, or using the [[html]] [^script^] element for JavaScripts). Although this
+					superficially makes the resources similar in terms of not requiring fallbacks, core media types are
+					differentiated by having stronger support expectations in reading systems.</p>
```

Comment:
Does this need a clarification about how such resources are defined? Something like "Resources of core media types that match an exemption remain core media types and are not reclassified as exempt resources." That is, a jpg referenced from a script is not an exempt resource, it remains a core media type. Or maybe that isn't true - I guess it depends if core media types can also be exempt, although we seem to consider these two distinct classes.

Reply:
Agree, the more we can clarify here the better. I've added a couple of additional sentences to make clear that CMTs do not become exempt resources because they share the same inclusion mechanisms and exempt resources without the exemption would be foreign resources. Hopefully, that improves it a bit.